index.html

```diff
@@ -88,126 +99,122 @@ <h3 class="title">Skills</h3>
             <a href=""><i class="fa-solid fa-m"></i></a>
           </li>
         </ul>
-        <div class="work">
+        <div class="work work-two">
           <div><i class="fa-solid fa-arrow-down"></i><span> Work</span></div>
         </div>
       </div>
     </section>
 
-    <!-- Middle part -->
-    <section class="middle-section">
-      <h3 class="section-title work-title">
-        MY RECENT <br />
-        WORK
-      </h3>
+    <!-- Main section -->
+    <section class="about" id="about">
+      <h2 class="color2">My recent work</h2>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_97](https://user-images.githubusercontent.com/69638013/165933678-ac4eeb77-5aec-4a40-96d2-6c1bba851439.png) | ![Screenshot_99](https://user-images.githubusercontent.com/69638013/165933979-8281b0ff-d146-4a17-92ee-c9a045affc24.png)

- [ ] Kindly align the header and the horizontal line to have the same width with the card below, please check screenshot above ⬆️